src/View/ImageLoader.py

```diff
@@ -16,10 +19,11 @@ class ImageLoader(QtCore.QObject):
 
     signal_request_calc_dvh = QtCore.Signal()
 
-    def __init__(self, selected_files, parent_window, *args, **kwargs):
+    def __init__(self, selected_files, existing_rtss_path, parent_window, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, selected_files, existing_rtss_path, parent_window, 
                 *args, **kwargs):
```

PEP 8 compliance - line too long (also lines 46-47, 134-137, 142-143).